solr/test-framework/src/java/org/apache/solr/embedded/JettySolrRunner.java

```diff
@@ -375,24 +370,11 @@ private void init(int port) {
           new ServletContextHandler(server, "/solr", ServletContextHandler.SESSIONS);
       root.setResourceBase(".");
 
-      server.addEventListener(
-          new LifeCycle.Listener() {
-
-            @Override
-            public void lifeCycleStopping(LifeCycle arg0) {}
-
-            @Override
-            public synchronized void lifeCycleStopped(LifeCycle arg0) {
-              if (coreContainerProvider != null) {
-                coreContainerProvider.close();
-              }
-            }
-
+      root.addEventListener(
+          new CoreContainerProvider() {
```

Comment:
Notice I'm subclassing CCP in order do some initialization

Reply:
maybe worth a comment in the code?